tests/test_rust.py

```diff
@@ -0,0 +1,9 @@
+from tests import unittest
```

Comment:
More of a style question / point to think about: do we want to write unit tests in Rust (and then use `cargo test` to run them in CI?)

Trial tests still seem to make sense for testing the external behaviour of the Rust module.